Cargo.toml

```diff
@@ -50,10 +44,12 @@ wasmparser = "0.227"
 # feature across all the transitive dependencies of kwctl
 # This is required to have kwctl use the system certificates instead of the
 # ones bundled inside of rustls
+indicatif = "0.17"
+is-terminal = "0.4.16"
 reqwest = { version = "0", default-features = false, features = [
   "rustls-tls-native-roots",
 ] }
-indicatif = "0.17"
+termimad = "0.31.2"
```

Comment:
Can you move the new dependecies above, like in the block that ends around line 47?

The current code messes up the comment that explains why we have this "strange" `reqwest` requirement around.

Reply:
Argh, with the rebase to main and the `indicatif` addition, I messed up with taplo.
Amended the commit.